### PR TITLE
Updated rules to match the new task 177 and 178 approach to versions

### PR DESCRIPTION
**Pull request by jsight**
Related PR: https://github.com/windup/windup/pull/993

### CODE REVIEW

File: rules-reviewed/eap6/commonj/commonj.windup.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
       </dependencies>
       <sourceTechnology id="weblogic" />
       <sourceTechnology id="websphere" />
-      <targetTechnology id="eap" versionRange="[6,)" />
+      <targetTechnology id="eap" versionRange="[6,7)" />
```
Comment (mareknovotny) [inline rules-reviewed/eap6/commonj/commonj.windup.xml]: so open range won't be supported?

**Comment by mareknovotny** [comment]
I think we need to update also maven-jar-plugin configuration to include '*.windup.technologytransformer.xml' files into the windup-rulesets jar artifact. Somewhere here https://github.com/windup/windup-rulesets/blob/master/pom.xml#L253-L257

**Comment by jsight** [comment]
Good point. I will take a look when I get back.

File: rules-reviewed/eap6/commonj/commonj.windup.xml (hunk 1 above)
Comment (mareknovotny) [inline rules-reviewed/eap6/commonj/commonj.windup.xml]: indeed, I just commented the first change as I noticed you changed all similar rules version ranges ;)

**Comment by jsight** [comment]
@mareknovotny Is this ok to merge as well?

**Comment by mareknovotny** [comment]
yes, I was only disrupted by the mtg we had yesterday ;)

**Comment by jsight** [comment]
Oh, I can understand that.

**Pull request merged**